templates/local.cfg.j2

```diff
@@ -193,7 +193,7 @@ handle.prefix = {{ dspace_handle_prefix }}
 # You may also copy default settings from the corresponding authentication-*.cfg
 # configuration file and override them in this local.cfg.
 
-# IP-based authentication/authorization. See authentication-ip.cfg for default configuration.
+# IP-based authentication/authorization. See authentication-ip.cfg for default configuration.:
```

Comment:
That looks suspiciously like a Vim-based typo. That's right, I blame Vim. I'll revise this.